setup-r-dependencies/action.yaml

```diff
@@ -18,7 +18,16 @@ inputs:
     description: 'Which package(s) to install.'
     default: 'deps::., any::sessioninfo'
   pak-version:
-    description: 'Which pak version to use. Possible values are "stable", "rc" and "devel".'
+    description: |
+      Which pak version to use. Possible values are "stable", "rc",
+      "devel", "none", "repo". The first three install pak from our
+      repository at GitHub Pages.
+      "none" will skip pak installation. Use this if you want to install
+      pak yourself. Set the `R_LIB_FOR_PAK` environment variable to point
+      to the library where pak is installed.
+      `repo` means that the action will install pak from the configured
+      repositories, using `install.packages()`. `repo` is appropriate on
+      systems that do not have access to our pak reporitory on GitHUb.
```

Comment:
So does `repo` imply cran in most cases?

Reply:
Whatever is set up. If you use `setup-r` then typically either PPM or CRAN.

Reply:
Maybe mention here that `repo` gives the cran version of pak by default ? I had to read it a few times to understand that `repo` means here, it is such an overloaded term.

Reply:
Well, it does not give the cran version, but it gives whatever you have in the configured repo. If you use the recommended workflows, it'll be installed from PPM on Linux.